ern-container-gen/src/generateContainer.js

```diff
@@ -43,7 +43,7 @@ export default async function generateContainer ({
   const COMPOSITE_MINIAPP_DIRECTORY = path.join(workingDirectory, 'compositeMiniApp')
 
   const paths : ContainerGeneratorPaths = {
-    containerHull: path.join(Platform.currentPlatformVersionPath, 'ern-container-gen', 'hull', generator.platform),
+    containerHull: path.join(__dirname, '..', 'hull', generator.platform),
     containerTemplates: path.join(Platform.currentPlatformVersionPath, 'ern-container-gen', 'templates', generator.platform),
```

Comment:
Could you please also update the path here the same way you did for `hull` directory, as the `templates` directory sits at the same level. To keep coherency. You can then also remove the `Platform` import as this line is the only usage left of it.

Reply:
Nevermind, I'll merge it and do this part of another PR, noticed some other cleanup that can be done following your refactoring and was working on a some cleanup work right now, so that will include that. Thanks again.

Reply:
:+1: Sounds good. Seeing some similar path-related problems with `ern run-android`. I can open another PR if needed.